04-onion-routing.md

```diff
@@ -575,7 +597,7 @@ HTLC at the final hop:
 2. data:
    * [`4`:`cltv_expiry`]
 
-If the `amt_to_forward` does not match the `incoming_htlc_amt` of
+If the `amt_to_forward` is higher than `incoming_htlc_amt` of
```

Comment:
@rustyrussell Actually now I think it is rather the opposite:the specs above say that a HTLC should be rejected if amt_to_forward > incoming_htlc_amt - fee. For the last hop, the specs also says that the fee is set to 0 so the rejection criteria becomes incoming_htlc_amt > amt_to_forward which is exactly what I wrote. If the last node tests for strict equality then it behaves differently from the other nodes